Libraries/CustomComponents/NavigationExperimental/NavigationCard.js

```diff
@@ -87,17 +87,15 @@ class NavigationCard extends React.Component<any, Props, any> {
       ...props,
     } = this.props;
 
-    if (style === undefined) {
-      // fall back to default style.
-      style = NavigationCardStackStyleInterpolator.forHorizontal(props);
-    }
+    const defaultStyle = NavigationCardStackStyleInterpolator.forHorizontal(props);
+
     if (panHandlers === undefined) {
       // fall back to default pan handlers.
       panHandlers = NavigationCardStackPanResponder.forHorizontal(props);
     }
 
     return (
-      <Animated.View {...panHandlers} style={[styles.main, style]}>
+      <Animated.View {...panHandlers} style={[styles.main, defaultStyle, style]}>
```

Comment:
Agree. We should add docs regarding this to avoid confusion. But overriding animation style should be possible.